cmd/cp.go

```diff
@@ -18,10 +19,45 @@ var CpCommand = &cobra.Command{
 		"Read from stdin: cat /path/to/file | qsctl cp - qs://prefix/stdin",
 		"Write to stdout: qsctl cp qs://prefix/b - > /path/to/file",
 	),
-	Args: cobra.ExactArgs(2),
-	RunE: cpRun,
+	Args:    cobra.ExactArgs(2),
+	RunE:    cpRun,
+	PreRunE: validateCpFlag,
 }
 
 func cpRun(_ *cobra.Command, args []string) (err error) {
 	return action.Copy(args[0], args[1])
 }
+
+func initCpFlag() {
+	CpCommand.PersistentFlags().StringVar(&expectSize,
+		expectSizeFlag,
+		"",
+		"expected size of the input file"+
+			"accept: 100MB, 1.8G\n"+
+			"(only used and required for input from stdin)",
+	)
+	CpCommand.PersistentFlags().StringVar(&maximumMemoryContent,
+		maximumMemoryContentFlag,
+		"",
+		"maximum content loaded in memory\n"+
+			"(only used for input from stdin)",
+	)
+}
+
+func validateCpFlag(_ *cobra.Command, _ []string) (err error) {
+	if expectSize != "" {
+		contexts.ExpectSize, err = utils.ParseByteSize(expectSize)
+		if err != nil {
+			return
+		}
+	}
+
+	if maximumMemoryContent != "" {
+		contexts.MaximumMemoryContent, err = utils.ParseByteSize(maximumMemoryContent)
```

Comment:
We need to find another way to transfer flag values, maybe in a context.

Use `contexts.MaximumMemoryContent` will make it harder to do unit test.